guides/release/community-release.md

```diff
@@ -0,0 +1,98 @@
+# GitHub Release Process
+
+## 1. Requirements
+
+a. **GitHub Token**
```

Comment:
I still continue finding the indentation too much. 
I would remove the letter based bullet point or the next bullet points.
All the entries only have one unique entry...
```
1.
b.
-
```
XD

Reply:
Fair point, i will remove one level: '-'

Reply:
and i removed the letters, same style as on the [hack on trento guide](https://github.com/trento-project/web/blob/main/guides/development/hack_on_the_trento.md)